src/cmds/swagger.ts

```diff
@@ -1,5 +1,5 @@
-import type { CommandOptions } from '../lib/baseCommand';
```

Comment:
lint 🤔 per [this thread](https://github.com/readmeio/rdme/pull/732#discussion_r1080568227) I'm not sure if this is a bug or not (per https://github.com/import-js/eslint-plugin-import/issues/2683) but I'm just gonna roll with this for now.